auau/augmenter.py

```diff
@@ -98,15 +98,25 @@ def pitch_scale(self, signal, sr, num_semitones=2):
         augmented_signal = FAudio.pitch_shift(signal, sr, num_semitones)
         return self._tensor_on_device(augmented_signal)
 
-    def random_gain(self, signal, min_gain=1, max_gain=1.2):
-        """Random gain, processing on device
+    def gain(self, signal, gain):
+        """
+        Multiply signal by gain. 
+        gain can either be a constant value, or a tensor with the same shape as signal
         """
         signal = self._tensor_on_device(signal)
         signal = self._check_and_convert_mono(signal)
-        gain_rate = torch.FloatTensor(signal.shape).uniform_(min_gain, max_gain)
```

Comment:
I'm addressing this in the new version of random_gain(), but since this is the old code, I'll comment it here:

This was not going to cuda before, thus crashing when running things on cuda